src/client/eppo-client-with-bandits.spec.ts

```diff
@@ -73,69 +73,46 @@ describe('EppoClient Bandits E2E test', () => {
   });
 
   describe('Shared test cases', () => {
-    const testData = readTestData<BanditTestCase>(BANDIT_TEST_DATA_DIR);
-    // Build a map for more useful test names
-    const testsByFlagKey: Record<string, BanditTestCase> = {};
-    testData.forEach((testCase) => (testsByFlagKey[testCase.flag] = testCase));
```

Comment:
Keying by flag is no longer good as we have multiple test cases / files that share share flag names. So now we key by file name.